hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2237,13 +2237,22 @@ public void deleteKey(OmKeyArgs args) throws IOException {
     }
   }
 
-  // TODO: HDDS-2424. recover-trash command server side handling.
   @Override
   public boolean recoverTrash(String volumeName, String bucketName,
       String keyName, String destinationBucket) throws IOException {
 
-    boolean recoverOperation = true;
-    return recoverOperation;
+    if (isAclEnabled) {
```

Comment:
We don't need this anymore. As OMTrashRecoverRequest is handling recover request.

Reply:
Thanks for the review.
Updated.